Server: Reload TLS certs without a server restart (#83589)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

Path to the certificate key file (if `protocol` is set to `https` or `h2`).

### certs_watch_interval

Controls whether `cert_key` and `cert_file` are periodically watched for changes.
Disabled, by default. When enabled, `cert_key` and `cert_file`

> are watched for changes, if there is change seen then those are loaded automatically.

Reviewer: ```suggestion
are watched for changes. If there is change, the new certificates are loaded automatically.
```

{{% admonition type="warning" %}}
After the new certificates are loaded, connections with old certificates

> will not work. You must reload the connections to the old certs for them to work. 

Reviewer 2: the lint checker hates trailing spaces
```suggestion
will not work. You must reload the connections to the old certs for them to work.
```
